### DISCUSSION TITLE
Alerts forward to SIEM via syslog?

### DISCUSSION THREAD

**Discussion by y0d4a** [Security-Onion-Solutions/securityonion#6373] [Unsupported Versions] [answered]
Hi,

how can i forward only alerts log to SIEM?
syslog should be fine?
but not sure which file/filters...

thank you

**Comment by tomjones1977** [comment]
Alerts from what?  We need more information than this to be able to help.

**Reply by y0d4a** [reply]
general, all alerts which i saw when open alerts?

**Comment by weslambert** [comment, marked as answer]
Please see: https://docs.securityonion.net/en/latest/logstash.html?highlight=forwarding#modified-event-forwarding
